src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -214,9 +214,9 @@ public override string ToString()
             // to differ only by upper/lower case.  If they do, that's really
             // a code bug, and the effect is to just display both strings.
 
-            return string.Equals(_punycodeName, _unicodeName) ?
-                        _punycodeName :
-                        _unicodeName + " (" + _punycodeName + ")";
+            return string.Equals(_punycodeName, _unicodeName, StringComparison.Ordinal)
+                ? _punycodeName
+                : _unicodeName + " (" + _punycodeName + ")";
```

Comment:
We use StringComparison.OrdinalIgnoreCase in other places.

Reply:
The preceding comment:

```
            // Use case sensitive comparison here.
            // We don't ever expect to see the punycode and unicode strings
            // to differ only by upper/lower case.  If they do, that's really
            // a code bug, and the effect is to just display both strings.
```

Reply:
@iSazonov The comment explains why we shouldn't use `StringComparison.OrdinalIgnoreCase`.